src/test/java/de/tum/in/ase/parser/IntegrationTest.java

```diff
@@ -16,69 +21,97 @@
+    private final static String EXPECTED_FOLDER_PATH = "src/test/java/expected/";
```

Comment:
Does this work on Windows, as it uses a different path separator? Using `Paths.get("src", "test", "java", "expected")` instead should definitely work.

Reply:
It should work with UNIX paths for Windows as well, since Java is usually able to convert paths without a problem. But using your suggestion should make it more robust :+1: